src/core/theme/colors.ts

```diff
@@ -88,7 +88,7 @@ export const shadows = {
   panelShadow: `0 1px 2px 0 ${alphaHex(0.14)(
     colors.blackBase,
   )}, 0 1px 5px 0 ${alphaHex(0.12)(colors.blackBase)}`,
-  dropdownShadow: `0 1px 2px 0 ${alphaHex(0.1)(colors.brandBase)} inset`,
+  inputBoxShadow: `0 1px 2px 0 ${alphaHex(0.1)(colors.brandBase)} inset`,
```

Comment:
Is dropdown / select considered an input? Input box shadow might not be a good name for this since it's also used for dropdown. What about action element box shadow? Form element box shadow? I will leave it for you to decide.

Reply:
Changed this to actionElementBoxShadow.